installation/prereq.adoc

```diff
@@ -0,0 +1,102 @@
+= Prerequisites
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc:
+:toc-placement!:
+:toc-title: Topics Covered
+
+toc::[]
+
+== Overview
+
+Whether installing OpenShift using RPM packages or Docker images, you must first satisfy the following prerequisites.
+
+== Installing Red Hat Enterprise Linux 7
```

Comment:
This topic is going to need to be ifdef'd. RHEL7 is an OSE requirement, but not an Origin requirement. See https://github.com/nhr/openshift-docs#distribution-specific-conditionals